ipatests/test_integration/test_commands.py

```diff
@@ -699,3 +699,61 @@ def is_tls_version_enabled(tls_version):
         assert not is_tls_version_enabled('tls1')
         assert not is_tls_version_enabled('tls1_1')
         assert is_tls_version_enabled('tls1_2')
+
+    def test_sss_ssh_authorizedkeys(self):
+        """
```

Comment:
Issues with the docstring:
1. It does not reflect the actual test case. Actual steps are: 1) setup user with ssh-key and certificate stored in ipaserver 2) simulate p11_child timeout 3) try to login via ssh using private key.
And the description is something like "Check private ssh key is retrieved when user has certificate and p11_child times out." - I guess that is the original bug summary.
2. As per PEP 257 [1], there should be no new line between opening quotes and first line.

[1] https://www.python.org/dev/peps/pep-0257/